docs/rules/prefer-map-has.md

```diff
@@ -0,0 +1,56 @@
+# Prefer `Map#has` over `Map#get`
+
+💼 This rule is enabled in the ✅ `recommended` [config](https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs-eslintconfigjs).
+
+💡 This rule is manually fixable by [editor suggestions](https://eslint.org/docs/latest/use/core-concepts#rule-suggestions).
+
+<!-- end auto-generated rule header -->
+<!-- Do not manually modify this header. Run: `npm run fix:eslint-docs` -->
+
+In Boolean expressions, prefer `Map#has` over `Map#get`. For example, in IfStatements, WhileStatements, DoWhileStatements, ForStatements, and ConditionalExpressions.
```

Comment:
I find it hard to believe this is the best you can do even with the help of AI.

This is what I got back, which is much better:

> Use `Map#has` instead of `Map#get` to check for the presence of a key in a `Map`. `Map#has` is more semantically correct for existence checks and avoids unnecessary retrieval of values.

Reply:
My bad, I didn't find the keywords